gcloud-java-dns/src/main/java/com/google/gcloud/dns/ZoneInfo.java

```diff
@@ -255,7 +232,7 @@ public Builder toBuilder() {
     pb.setDescription(this.description());
     pb.setDnsName(this.dnsName());
     if (this.id() != null) {
-      pb.setId(this.id());
+      pb.setId(new BigInteger(this.id()));
```

Comment:
Fine as is. An alternative would be to keep it internally as BigInteger, however return it to the user as string (id.toString()).